src/views/Verify.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div>
+    <h1>You have to verify your email!</h1>
+    <button @click="singOut">SING OUT</button>
```

Comment:
There is a typo. It should be `Sign out`.

Reply:
Thanks man :D I changed it